src/liboslexec/batched_backendllvm.cpp

```diff
@@ -800,40 +804,56 @@ BatchedBackendLLVM::llvm_load_mask(const Symbol& cond)
 llvm::Value*
 BatchedBackendLLVM::llvm_load_value(llvm::Value* ptr, const TypeSpec& type,
                                     int deriv, llvm::Value* arrayindex,
-                                    int component, TypeDesc cast,
-                                    bool op_is_uniform, bool index_is_uniform,
+                                    int component, bool ptr_is_uniform,
```

Comment:
`bool ptr_is_uniform`

I found the name to be confusing, because varying pointers exist and this is really saying what the ptr points to is uniform.

In the store I saw dst_is_uniform and it made total sense, so maybe for the loads we could just use 
`bool src_in_uniform`
or something instead

Reply:
Resolved in the new PR.